source/crud/tests/v2/aggregate-merge.yml

```diff
@@ -0,0 +1,136 @@
+data:
+    - { _id: 1, x: 11 }
+    - { _id: 2, x: 22 }
+    - { _id: 3, x: 33 }
+
+minServerVersion: '4.1.12'
+
+collection_name: &collection_name 'test_aggregate_merge'
+
+tests:
```

Comment:
Can we add tests that assert both $out and $merge are treated as writes, e.g. by specifying a non-default write concern and including the write concern in the command started event?

Reply:
At present, I don't believe the CRUD spec has a reliable way to specify these options, and it seems odd to add such tests to the transactions spec (it's runner would support this). I've opened [SPEC-1316](https://jira.mongodb.org/browse/SPEC-1316) to track this request.